modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase

```diff
@@ -55,6 +55,83 @@ definition {
 			name = "Channels");
 	}
 
+	@description = "This is a test for COMMERCE-10610. As a buyer user, I can view the default channels with no results using the view permissions for products and catalogs of the admin panel"
```

Comment:
Based on my following comment, please adjust the description accordingly. Thanks!